### PR TITLE
New Typings: Alexa voice service

### PR DESCRIPTION
**Pull request by dolanmiu**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by dolanmiu** [comment]
I need help with this. in the `alexa-voice-service-tests.ts has:

```ts
const avsInstance = new AVS(options);
```

It is red, and I am not sure how to resolve

File: types/alexa-voice-service/alexa-voice-service-tests.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const options = {
+    debug: true,
+    clientId: "",
+    clientSecret: "",
+    deviceId: "",
+    refreshToken: "",
+};
+
+const avsInstance = new AVS(options);
```
Comment (kacepe) [inline types/alexa-voice-service/alexa-voice-service-tests.ts]: I think `AVS` doesn't have default export so you should use `new AVS.AVS(options);`

**Review by kvcpr — CHANGES_REQUESTED** [review]

**Comment by dolanmiu** [comment]
It does have new AVS(options);

look at the official docs:

https://www.npmjs.com/package/alexa-voice-service

**Comment by aozgaa** [comment]
Please fix failing tests.

**Comment by aluanhaddad** [comment]
This should be in UMD style based on https://github.com/miguelmota/alexa-voice-service.js/blob/master/index.js

File: types/alexa-voice-service/alexa-voice-service-tests.ts (hunk 1 above)
Comment (aluanhaddad) [inline types/alexa-voice-service/alexa-voice-service-tests.ts]: Actually, this syntax does not imply the use of a `default`.

**Comment by dolanmiu** [comment]
> This should be in UMD style based on https://github.com/miguelmota/alexa-voice-service.js/blob/master/index.js

How do I make the typings UMD style? Can you point me in the right direction?

**Comment by aluanhaddad** [comment]
@dolanmiu actually, I commented on your open issue on this repository explaining (I hope) just how to do that.

**Comment by dolanmiu** [comment]
@aluanhaddad thanks for your help! it worked wonderfully.

**Pull request merged**